src/applications/disability-benefits/all-claims/containers/ITFWrapper.jsx

```diff
@@ -61,6 +69,27 @@ export class ITFWrapper extends React.Component {
   }
 
 
+  hijackScroller() {
+    // RoutedSavableApp scrolls to the top of the page title, but we want to scroll to the top of
+    //  the ITF message
+    Scroll.Events.scrollEvent.register('begin', () => {
+      // We only want to hijack the scrolling one time
+      Scroll.Events.scrollEvent.remove('begin');
+      if (!this.state.hasDisplayedSuccess) {
+        setTimeout(() => {
```

Comment:
Oh, meant to comment on that. Thanks for asking.

I can only guess. :confused:

It wouldn't work without the timeout, and it wouldn't work if the timeout was `undefined` or `100`. I think this is because it has to wait for the scroll that triggered this begin event to _actually_ start scrolling before overriding it with the new scroll to the ITF message. I was hoping to get away with just using the event listener, but that wasn't enough. :disappointed:

This also means that I can't guarantee `200` is enough for all devices. It _seems_ safe, wrapped in the event listener callback, but we can't extend that time too much, since the scroll it's overriding has a duration of 500 ms.